lib/incito-browser/incito.js

```diff
@@ -76,7 +76,9 @@ export default class Incito extends MicroEvent {
 
     start() {
         this.el.addEventListener('click', (e) => {
-            const link = e.target.getAttribute('data-link');
+            const link = e.target.closest
+                ? e.target.closest('[data-link]').getAttribute('data-link')
+                : e.target.getAttribute('data-link');
```

Comment:
Replying here instead of on the DM ;)

`@babel/preset` & `core-js` does not include polyfills for specific DOM features like this, I would propose a little something like this: (It's the MDN recommended polyfill changed to be a ponyfill instead, so we don't patch prototypes on client websites)

```js
const matches =
    Element.prototype.matches ||
    Element.prototype.msMatchesSelector ||
    Element.prototype.webkitMatchesSelector;

function closest(el, s) {
    do {
        if (matches.call(el, s)) return el;

        el = el.parentElement || el.parentNode;
    } while (el !== null && el.nodeType === 1);

    return null;
}
```

used like

```js
const link = closest(e.target, '[data-link]').dataset.link;
```

But do be mindful that `closest` can return null and crash this example and yours.

Reply:
Additionally it might be worth checking that the `[data-link]` element is inside the incito, in case integrators use data-link on their own site for whatever reason: 
```js
const link = closest(e.target, '.incito__view [data-link]').dataset.link;
```

Reply:
@klarstrup Lovely feedback. Much appreciated. I should have addressed it now.